content/courses/mobile/solana-mobile-dapps-with-expo.md

```diff
@@ -569,98 +620,98 @@ defaultConfig.resolver.extraNodeModules = {
   url: require.resolve("react-native-url-polyfill"),
   zlib: require.resolve("browserify-zlib"),
   path: require.resolve("path-browserify"),
+  crypto: require.resolve("expo-crypto"),
 };
 
 // Export the modified configuration
-module.exports = defaultConfig;
+module.exports = {
+  ...defaultConfig,
+  // See more why we have to do here at: https://github.com/metaplex-foundation/umi/issues/94
+  resolver: {
+    ...defaultConfig.resolver,
+    unstable_enablePackageExports: true,
+  },
+};
 ```
 
 #### 3. Metaplex provider
 
-We're going to create a Metaplex provider file that will help us access a
-`Metaplex` object. This `Metaplex` object is what gives us access to all of the
-functions we'll need like `fetch` and `create`. To do this we create a new file
-`/components/MetaplexProvider.tsx`. Here we pipe our mobile wallet adapter into
-an `IdentitySigner` for the `Metaplex` object to use. This allows it to call
-several privileged functions on our behalf:
+We're going to create a Metaplex provider file that will help us access an `Umi`
+object (Read more about `umi` at
+[Umi docs](https://developers.metaplex.com/umi)).This `Umi` object, combined
+with other libraries such as `@metaplex-foundation/umi-bundle-defaults`,
+`@metaplex-foundation/mpl-token-metadata`and
+`@metaplex-foundation/mpl-candy-machine`, will give us access to all the
+functions we'll need later, like `fetch` and `create`.. To do this we create a
+new file `/components/MetaplexProvider.tsx`. Here we pipe our mobile wallet
+adapter into the `Umi` object to use. This allows it to call several privileged
+functions on our behalf:
```

Comment:
Good work, but focus on token-metadata more than Umi. 

Ie something like

> We'll be creating NFTs using [Metaplex's MPL Token Metadata](https://developers.metaplex.com/token-metadata) library, via [Umi](https://developers.metaplex.com/umi) a tool that is used in many Metaplex apps. 

Basically the Umi docs are weird and confusing and don't mention making NFTs,